storyblok/storyblok-design-system
# refactor: sbBreadcrumbs to template syntax

## Pull request type

Jira Link: -




## How to test this PR


Compare:
* https://blok.ink/?path=/story/design-system-components-sbbreadcrumbs--default
*  https://storyblok-design-system-git-refactor-ar-11-e181be-storyblok-com.vercel.app/

It's also good to check in the app:

* `cd storyblok-design-system && git checkout refactor/ar-11-breadcrumbs`
* Then start the server
## What is the new behavior?



- uses the template syntax
- 
- 

## Other information
